tests/acceptance/test_organization_events_v2.py

```diff
@@ -52,12 +60,54 @@ def test_all_events(self, mock_now):
         )
 
         with self.feature(FEATURE_NAMES):
-            self.browser.get(self.path + "?" + all_view)
+            self.browser.get(self.path + "?" + all_events_query)
+            self.wait_until_loaded()
+            self.browser.snapshot("events-v2 - all events query - list")
+
+    @patch("django.utils.timezone.now")
+    def test_modal_from_all_events_query(self, mock_now):
+        mock_now.return_value = before_now().replace(tzinfo=pytz.utc)
+        min_ago = iso_format(before_now(minutes=1))
+
+        event_data = load_data("python")
+        event_data.update(
+            {
+                "event_id": "a" * 32,
+                "timestamp": min_ago,
+                "received": min_ago,
+                "fingerprint": ["group-1"],
+            }
+        )
+        event = self.store_event(
+            data=event_data, project_id=self.project.id, assert_no_errors=False
+        )
+
+        with self.feature(FEATURE_NAMES):
+            # Get the list page.
+            self.browser.get(self.path + "?" + all_events_query)
+            self.wait_until_loaded()
+
+            # Click the event link to open the modal
+            self.browser.element('[aria-label="{}"]'.format(event.title)).click()
+            self.wait_until_loaded()
+
+            header = self.browser.element('[data-test-id="modal-dialog"] h2')
+            assert event_data["message"] in header.text
+
+            issue_link = self.browser.element('[data-test-id="linked-issue"]')
```

Comment:
There is a helper method on .browser that let's you select elements by their test id.

Reply:
I cannot find this method. 🤔